.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: build
+on: [ push, pull_request ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.4', '2.5', '2.6', '2.7' ]
+        gemfile: [ 'rails_5_0', 'rails_5_1', 'rails_5_2', 'rails_6_0' ]
+        exclude:
+          # Only test latest Rails (Of each major), on actively supported Ruby Versions
+          # Only test latest Ruby with latest Rails versions (Of each major)
+          # 2.4  -> Not 5.2/6.0
+          # 2.5  -> Anything
+          # 2.6  -> Users of this should be using Rails 5.2+
+          # 2.7  -> Users of this should be using Rails 5.2+
+          - ruby: '2.4'
+            gemfile: 'rails_5_2'
+          - ruby: '2.4'
+            gemfile: 'rails_6_0'
+          - ruby: '2.6'
+            gemfile: 'rails_5_0'
+          - ruby: '2.6'
+            gemfile: 'rails_5_1'
+          - ruby: '2.7'
+            gemfile: 'rails_5_0'
+          - ruby: '2.7'
+            gemfile: 'rails_5_1'
+    env:
+      BUNDLE_GEMFILE: gemfiles/${{ matrix.gemfile }}.gemfile
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
```

Comment:
I'm not sure of what node version we use in our repo. But if at all possible can we use fermium (14)

Reply:
Added in [`b0fb685`](https://github.com/cucumber/cucumber-rails/compare/51ac21f81d8fdcb88d49c699e38c19d9f0a2f30b..b0fb68513929b07ca88cd624da6b4007017dc13e)